src/System.Data.Odbc/tests/Configurations.props

```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <BuildConfigurations>
```

Comment:
That line in your project file is actually dead code because you don't have a netfx configuration for this library at all. 

The issue with the netfx vertical is that your test project didn't define any configurations and as such it tries to build for all configurations, but when it tries to build for netfx there are type conflicts because you duplicate some types in your library with things that live in System.Data in netfx.

I didn't get to see what your other errors were but I do think you need to add a configurations.props for your test project. You might need to make some tweaks to that test project once you add the configuration. Make sure you just try and build the test project once you add that and get that building and then CI should be good hopefully.